packages/flutter/lib/src/widgets/autocomplete.dart

```diff
@@ -502,6 +505,30 @@ class _RawAutocompleteState<T extends Object> extends State<RawAutocomplete<T>>
     };
     _updateActions();
     _updateOverlay();
+
+    WidgetsBinding.instance.addObserver(this);
+    WidgetsBinding.instance.addPostFrameCallback((_) => _getFieldWidth());
+  }
+
+  bool _isOffstage = true;
+  double? _fieldWidth;
+  final GlobalKey _textFieldKey = GlobalKey();
+
+  void _getFieldWidth() {
```

Comment:
Nit: This name makes me think it's going to return the width.  Maybe `_updateFieldWidth` or `_updateSizing` or something?